lib/data/models/activity_model.dart

```diff
@@ -8,11 +8,11 @@ class ActivityModel {
   /// Use startTime as unique identifier.
   Id get id => startTime.microsecondsSinceEpoch;
 
-  DateTime startTime = DateTime.fromMillisecondsSinceEpoch(0);
+  late DateTime startTime;
```

Comment:
I honestly don't know either, but I though it was a better idea to actually *require* values rather than putting default ones. If it does not work we'll put default values anyway :)